### PR TITLE
Add http:// to user_guide/runtime-conf.md

### PR DESCRIPTION
**Pull request by ca-scribner**
Current example does not work.  Minio endpoint requires http://, otherwise you get an invalid endpoint error.

Sorry didn't have a chance to check development outlines, etc.  Hopefully with such a minor change nobody minds (or they implement it however they'd prefer) :)
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: docs/source/user_guide/runtime-conf.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ elyra-metadata install runtimes --schema_name=kfp \
        --name=my_kfp \
        --display_name="My Kubeflow Pipeline Runtime" \
        --api_endpoint=https://kubernetes-service.ibm.com/pipeline \
-       --cos_endpoint=minio-service.kubeflow:9000 \
+       --cos_endpoint=http://minio-service.kubeflow:9000 \
```
Comment (lresende) [inline docs/source/user_guide/runtime-conf.md:38]: could you please also update the "result json" right below.

File: docs/source/user_guide/runtime-conf.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ elyra-metadata install runtimes --schema_name=kfp \
        --name=my_kfp \
        --display_name="My Kubeflow Pipeline Runtime" \
        --api_endpoint=https://kubernetes-service.ibm.com/pipeline \
```
Comment (lresende) [inline docs/source/user_guide/runtime-conf.md:37]: Looking into [recipes/deploying-kubeflow-locally-for-dev.md](https://elyra.readthedocs.io/en/latest/recipes/deploying-kubeflow-locally-for-dev.html), I was wondering if we should use the same pattern here as if you were deploying it locally and use `--api_endpoint=http://localhost:31380/pipeline`

**Comment by ca-scribner** [comment]
oh yeah sorry didn't even read the json output in the next paragraph.  whoops.

File: docs/source/user_guide/runtime-conf.md (hunk 2 above)
Comment (ca-scribner) [reply docs/source/user_guide/runtime-conf.md:37]: That makes sense to me, but I can't say if localhost:31380 actually works (I have not run through the other examples - just using an existing kfp).  It does sound a lot more likely to work in an example than `kubernetes-service.ibm.com` though

**Pull request merged**